app/models.py

```diff
@@ -96,3 +98,84 @@ def facet_plural(facet: str):
         facet_plural = facet
 
     return facet_plural
+
+
+# --------------------------------------------
+# Response model class for the knowledge panels
+# --------------------------------------------
+
+
+class ElementsForDataQualityAndLastEdits(BaseModel):
+    element_type: str
+    text_element: Optional[str] = None
+
+
+class ElementsForHungerGame(BaseModel):
+    id: int
+    element_type: str
+    text_element: str
```

Comment:
You are describing a single element don't use plural here…

```python
class HungerGameElement(TextFacet):
    id: int
```

Reply:
@alexgarel If I do this the response will look like this(showing id below "element_type" and "text_elements") honestly  which I don't like that much but if you ask I will change it
<img width="237" alt="image" src="https://user-images.githubusercontent.com/96618001/192774306-44125bce-5380-4e06-bb73-83adc36488f0.png">

Reply:
Another thing is if I inherit  "TextFacet"  in  "WikidataPanel" then we lost the pydantic magic and the response became like this
<img width="363" alt="image" src="https://user-images.githubusercontent.com/96618001/192775048-9fbf0010-7e6c-4407-ae51-e4265129ae95.png">
So, my proposal is kept "TextFacetWikiData", what are your thoughts?

Reply:
@sumit-158
1. order is not important at all
2. I don't understand you point about pidantic !

Reply:
1. okk I will do it!
2. see https://github.com/openfoodfacts/facets-knowledge-panels/pull/24#discussion_r982381639